server/IDEs/eclipse/Refine with extensions.launch

```diff
@@ -1,18 +1,18 @@
-<booleanAttribute key="org.eclipse.jdt.launching.ATTR_USE_CLASSPATH_ONLY_JAR" value="false"/>
-<stringAttribute key="org.eclipse.jdt.launching.CLASSPATH_PROVIDER" value="org.eclipse.m2e.launchconfig.classpathProvider"/>
-<stringAttribute key="org.eclipse.jdt.launching.MAIN_TYPE" value="com.google.refine.Refine"/>
-<stringAttribute key="org.eclipse.jdt.launching.PROJECT_ATTR" value="server"/>
-<stringAttribute key="org.eclipse.jdt.launching.SOURCE_PATH_PROVIDER" value="org.eclipse.m2e.launchconfig.sourcepathProvider"/>
-<stringAttribute key="org.eclipse.jdt.launching.VM_ARGUMENTS" value="-Drefine.autoreloading=true -Dbutterfly.modules.path=modules,&quot;${workspace_loc:extensions}&quot;,&quot;${workspace_loc:jython}&quot;,&quot;${workspace_loc:extensions}&quot;,&quot;${workspace_loc:database}&quot;"/>
-<stringAttribute key="org.eclipse.jdt.launching.WORKING_DIRECTORY" value="${workspace_loc:OpenRefine}"/>
+    <listAttribute key="org.eclipse.debug.core.MAPPED_RESOURCE_PATHS">
+        <listEntry value="/server/src/com/google/refine/Refine.java"/>
+    </listAttribute>
+    <listAttribute key="org.eclipse.debug.core.MAPPED_RESOURCE_TYPES">
```

Comment:
Changes to this file (and to `main/webapp/package-lock.json` are unrelated to this PR, I would keep them out.

Reply:
Okay I will do that! 

Thank you for the review!

Reply:
cc @TheTrailblazer09 there is a review to address.